Clarify how to enable opinionated warnings

The section does not mention that ignore has to be overridden, which was non-obvious to me. I was attempting to use extend-ignore for E501, and so never saw B950 warnings because it was still in the ignore list as added by flake8-bugbear.

The README says:

> To enable these checks, specify a `--select` command-line option or
`select=` option in your config file.  As of Flake8 3.0, this option
is a whitelist (checks not listed are being implicitly disabled), so you
have to explicitly specify all checks you want enabled. For example:
>
> ```
> [flake8]
> max-line-length = 80
> max-complexity = 12
> ...
> select = C,E,F,W,B,B901
> ```

Example (no output from flake8):
```
$ pip install flake8 flake8-bugbear
$ printf '#%.0s' {0..100} > test.py
$ flake8 --select E,B9 --extend-ignore=E501 test.py
```

ignore must be set before there is any output:

```
$ flake8 --select E,B9 --ignore=E501 test.py
test.py:1:101: B950 line too long (100 > 79 characters)
```

--- a/README.rst
+++ b/README.rst
@@ -202,11 +202,14 @@ have to explicitly specify all checks you want enabled. For example::
 	max-line-length = 80
 	max-complexity = 12
 	...
+	ignore = E501
 	select = C,E,F,W,B,B901
 
 Note that we're enabling the complexity checks, the PEP8 ``pycodestyle``
 errors and warnings, the pyflakes fatals and all default Bugbear checks.
 Finally, we're also specifying B901 as a check that we want enabled.
+Some checks might need other flake8 checks disabled - e.g. E501 must be
+disabled for B950 to be hit.
 
 If you'd like all optional warnings to be enabled for you (future proof
 your config!), say ``B9`` instead of ``B901``. You will need Flake8 3.2+

